task/ruby-lint/0.1/README.md

```diff
@@ -0,0 +1,82 @@
+# Ruby Linter
+
+The following task is used to provide static analysis on YAML files mounted using `rubocop` (Ruby linter).
+
+## Installing the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/ruby-lint/0.1/rubocop-lint.yaml
```

Comment:
Can you correct the yaml file name here please, it doesn't match the yaml file name